### PR TITLE
fix: tooltip updates and loading spinner on navigate

### PR DESCRIPTION
**Pull request by cabreraalex**
# Description

Switching between explore and home wasn't working well. ALso added spinner to header whenever navigating to show if it's being slow after you clicked (necessary after removing view transitions)

### CODE REVIEW

File: frontend/src/lib/util/tooltip.ts [hunk 1]
```diff
@@ -1,31 +1,43 @@
 import { tooltipState } from '$lib/stores';
 
 export function tooltip(node: HTMLElement, params: { text: string }) {
-	function handleFocus(e: MouseEvent) {
-		tooltipState.set({ hover: true, mousePos: { x: e.clientX, y: e.clientY }, text: params.text });
+	let isHovering = false;
```
Comment (Sparkier) [inline frontend/src/lib/util/tooltip.ts:4]: Why this change? Before we were not adding listeners that are not needed. Now we call the move function even if we're not hovering. Seems unnecessary.

Comment (cabreraalex) [reply frontend/src/lib/util/tooltip.ts]: Ah good point. Changed it back but still in a way that the tooltip text updates.

**Review by Sparkier — APPROVED** [review]
